tests/e2e/matmul/BUILD.bazel

```diff
@@ -561,6 +565,10 @@ iree_generated_e2e_runner_test(
         "--shapes=gpu_large_aligned",
         "--compilation_info=LLVMGPUVectorDistributeWMMA",
     ],
+    runner_args = [
+        "--require_exact_results=false",
+        "--acceptable_fp_delta=1e-04",
+    ],
     tags = [
         "noasan",
         "nomsan",
```

Comment:
These tests should be removed from BUILD.bazel.

Reply:
Do you mean **noasan** and **nomsan**?

Reply:
Ok, you are actually asking to remove the entire test from Bazel.